src/renderer/registration/RegistrationTable.tsx

```diff
@@ -80,30 +80,34 @@ export default function RegistrationTable() {
             </TableRow>
           </TableHeader>
           <TableBody>
-            {models.map((model: MLModel) => (
-              <TableRow key={model.uid} className="py-2 hover:bg-gray-50">
-                <TableCell className="pl-4">{model.name}</TableCell>
-                <TableCell className="">
-                  {serverMap[model.uid]
-                    ? serverMap[model.uid].serverAddress
-                    : ''}
-                </TableCell>
-                <TableCell className="">
-                  {serverMap[model.uid] ? serverMap[model.uid].serverPort : ''}
-                </TableCell>
-                <TableCell className="">
-                  <div className="pl-4">
-                    <ModelStatusIndicator modelUid={model.uid} />
-                  </div>
-                </TableCell>
-                <TableCell>
-                  <ModelConnectionButton
-                    mutate={mutateServers}
-                    modelUid={model.uid}
-                  />
-                </TableCell>
-              </TableRow>
-            ))}
+            {models
+              .filter((model) => !model.isRemoved)
```

Comment:
instead of this, should we simply delete the row in the model server table when the model is removed?

Reply:
imo this is fine as well. @ShriramG24 thoughts?

Reply:
We can delete the row. Doesn't matter too much.

Reply:
We are using the models array to map the server info here so need to filter anyway I think.